build/e2e-test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
yes, ideally, but looks like does not work due to some travis issues -- need some investigation